README.md

```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
 [nREPL]: https://github.com/clojure/tools.nrepl
 [Arcadia Unity]: https://github.com/arcadia-unity/Arcadia
 
-## How to use
-TBD
+### How to use Cursive with Arcadia
+
+In your Unity project with Arcadia, do:
+
+1. Create a new "regular" looking clojure project that has a deps.edn at its root. For example:
+
```

Comment:
Perhaps we can explain what this is doing, and use proper syntax highlighting so it's a little clearer what is intended? E.g something like:

Create a `deps.edn` file with the following: 
```Clojure
{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
        arcadia             {:local/root "../Arcadia"}}}
```